src/content/references/en/8-climate-polygons.md

```diff
@@ -0,0 +1,15 @@
+---
+image: ../../../images/references/climate-polygons.png
+title: Reporting of areas with environmental data
+---
+
+Tasks such as regular reporting of regional weather and climate metrics, e.g. on water availability, often require a lot of manual work to process the weather data up and downstream. Here we have implemented a workflow that can completely automate this task.
+
+## Highlights
+
+- Automation of reporting: calculation of temporal aggregates for regions
+- Integration of region areas (polygons) as well as weather and climate data
+
+## Methodik
```

Comment:
should be Methodology

Reply:
[304b434](https://github.com/geo-engine/geoengine.de/pull/6/commits/304b4343dd4c0ea1ab9fdb71988815b9b1b5c145)